docs/index.html

```diff
@@ -69,7 +69,7 @@ <h2>Topics</h2>
 </div> <!-- id="main" -->
 <div id="about">
 <i>generated by <a href="http://github.com/stevedonovan/LDoc">LDoc 1.4.6</a></i>
-<i style="float:right;">Last updated 2018-07-18 10:19:19 </i>
+<i style="float:right;">Last updated 2018-07-18 10:44:53 </i>
```

Comment:
Can you remove the rendered documentation from the commits? I think (or at least I try) to only render docs upon a release commit, which also updates the version numbers etc.